camelAssemblies/README.md

```diff
@@ -78,6 +78,25 @@ The Vantiq Camel assemblies contained herein are composed of the following.
   * Each service is defined with an _inbound_ or _outbound_ event named with the base name with the suffix 
     `_serviceEvent`. Continuing the example above, `aws_sns_sink_serviceEvent` (so the full service event name is 
     `com.vantiq.extsrc.camel.kamelets.v3_21_0.aws_sns_sink.aws_sns_sink_service/aws_sns_sink_serviceEvent`).
```

Comment:
Outline how deployment works (that is, the procedure we have for the user to use to perform a K8sCluster/K8sInstallation deployment